projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -16,6 +16,13 @@ class Jetpack_Podcast_Helper {
 	 */
 	protected $feed = null;
 
+	/**
+	 * The total number of items
+	 *
+	 * @var number
```

Comment:
Should it be `int` here?

```suggestion
	 * @var int
```